Accelerated_Python_User_Guide/notebooks/Chapter_11_Distributed_Computing_cuPyNumeric.ipynb

```diff
@@ -53,7 +53,24 @@
    "source": [
     "## Basic installation\n",
     "\n",
-    "### Local linux server\n",
+    "### Local Linux server (pip)\n",
+    "\n",
+    "Python wheel packages for Python 3.10, 3.11, and 3.12 are available for AMD64 and ARM64 from [PyPI](https://pypi.org/project/nvidia-cupynumeric). To install these into an existing environment, run the command\n",
+    "```sh\n",
+    "pip install nvidia-cupynumeric\n",
+    "```\n",
+    "\n",
```

Comment:
```suggestion
    "\n",
    "or create a new environment:\n",
    "\n",
```

Reply:
Is something missing here? I get:

```
-> % pip install nvidia-cupynumeric
Looking in indexes: https://pypi.org/simple, https://pypi.ngc.nvidia.com
ERROR: Ignored the following yanked versions: 0.0.0
ERROR: Could not find a version that satisfies the requirement nvidia-cupynumeric (from versions: none)
ERROR: No matching distribution found for nvidia-cupynumeric

```

Reply:
When I switch to a different server, I get an error installing this way:

```
aterrel@e22b0d3-lcedt:~$ pip install nvidia-cupynumeric                                                                                                                                         21:48:48 [60/60]
Defaulting to user installation because normal site-packages is not writeable               
Collecting nvidia-cupynumeric                                                                           
  Downloading nvidia_cupynumeric-25.3.1-cp312-cp312-manylinux_2_28_x86_64.whl.metadata (924 bytes)    
...
aterrel@e22b0d3-lcedt:~$ vi black_sholes.py
aterrel@e22b0d3-lcedt:~$ legate --cpus 1 --sysmem 10000 ./black_sholes.py --num 10000 --precision 32 --time
legate: command not found
```

Reply:
using a virtualenv I get farther, but hit other bugs.

Reply:
Opened a follow-up issue on https://github.com/nv-legate/cupynumeric.internal/issues/684

Reply:
What Python version is being used? We only have packages for Python 3.10, 3.11 and 3.12 on x86_64 and aarch64.

Reply:
Not sure, I tested by `pip` installing into a conda environment and did not have any issues:
```
dev312 ❯ which legate
/home/bryan/anaconda3/envs/dev312/bin/legate

dev312 ❯ legate
Python 3.12.8 | packaged by conda-forge | (main, Dec  5 2024, 14:24:40) [GCC 13.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

Reply:
@cryos I was specifically testing installing pip packages, which is why I pip installed them. I installed them into a conda env because I only ever use conda env. It was just a test to see if the instructions worked, and they did.

Reply:
Yes, I realize my suggestion was naive, and I revise my suggestion. I didn't realize the bad experience users face when wheels are not available for their machine. We should follow @aterrel's advice and explicitly note Linux, x86 + aarch64, python 3.10-3.12.

Reply:
https://github.com/NVIDIA/accelerated-computing-hub/pull/62/commits/3b9ca8c2d5766f4e07480de5ff4ffb8123325974